### ISSUE TITLE
Node documentation allows resizing textareas 

### ISSUE THREAD

**Issue by joeyballentine** [chaiNNer-org/chaiNNer#2175]
With no way to revert it either.

![image](https://github.com/chaiNNer-org/chaiNNer/assets/34788790/e727849f-3667-4377-95cb-e1ac850efb4f)

**Comment by RunDevelopment** [comment]
That's intentional.

> With no way to revert it either.

Use the horizontal scroll bar at the bottom of the pop over, scroll to the right, and then drag the handle to shrink the text box.

Or just select a different node page and then go back.

**Comment by joeyballentine** [comment]
Should it be intentional though? At the very least I think we should set a max size on it

**Comment by RunDevelopment** [comment]
> Should it be intentional though? 

Yeah, example nodes should behave the same way as real nodes.

> At the very least I think we should set a max size on it

Good point. Should the max size only affects example nodes or all nodes? Setting a max for all inputs might not be a bad idea.

**Comment by joeyballentine** [comment]
> Yeah, example nodes should behave the same way as real nodes.

They don't though, because you can't connect them to anything. The resizing doesn't really make sense when the space allotted to it isn't an infinite canvas but rather a finite sized modal. It just seems like a bug (which in my opinion it is)

**Comment by RunDevelopment** [comment]
> They don't though, because you can't connect them to anything.

As I said, "should". Behaving like real ones is the goal. Yes, they don't behave 100% like real ones, but it's close.

> The resizing doesn't really make sense when the space allotted to it isn't an infinite canvas but rather a finite sized modal.

It does. Our text boxes are quite small by default. Being able to make them bigger is useful even in examples.

**Comment by joeyballentine** [comment]
I'm highly skeptical of how useful it is to expand an example texbox. Exactly how does it improve the user experience to be able to type more into a box that does nothing?

**Comment by stonerl** [comment]
If I might add my 2 cents. All nodes in the documentation allow to be interacted with like “normal” nodes. E.g., the load image node actually allows selecting and loading an image. TBH, I don't see any benefit in this at all.

From my humble perspective, the nodes in documentation shouldn't be interactive and rather be like images, to show the users the corresponding node to the description.

**Comment by joeyballentine** [comment]
That's how they used to be. We thought it would be nice to allow the user to play around with them in the docs, i.e. to see what happens under certain conditions and what kind of values you can enter. I just don't think that same logic applies to resizing a textbox

**Comment by RunDevelopment** [comment]
> shouldn't be interactive

They kinda have to be because of conditional inputs. If you show just one "image", then you could be hiding most inputs.

Well, the main reason why I wanted them to be interactive is that I wanted node examples directly in the docs. So e.g. the Text Replace node would list a few examples of how the node behaves on various inputs and then users could play around with it to get a better understanding of it. We haven't added support for inline-examples yet, though.

**Comment by RunDevelopment** [comment]
> Exactly how does it improve the user experience to be able to see what they type in a larger box that does nothing?

True. We don't have large text boxes which values are used in the output type right now. It doesn't mean that we never will have nodes like this though.

Also, if you disable resizing for all inputs/outputs, then you are not showing one feature of the node. Especially for nodes like View Image, resizing will be one of its features (#2167). IMO, showing everything a node can do is just consistent.

**Comment by joeyballentine** [comment]
I just don't think the tiny little example window is the place for that though. Maybe if it expanded into a little react flow modal where you could make a little temp chain, then I could understand it. But again when I saw I could resize the textarea it just felt like a bug to me. It just doesn't look or feel good to resize those there

**Comment by stonerl** [comment]
> That's how they used to be. We thought it would be nice to allow the user to play around with them in the docs, i.e. to see what happens under certain conditions and what kind of values you can enter. I just don't think that same logic applies to resizing a textbox

Kk, I see.

**Comment by joeyballentine** [comment]
The other thing i'll mention with this, is it doesn't even work how it does in the editor. Since it's right aligne, when you resize, it expands in the direction opposite where you expand it to. It's extremely weird
